test-libzstd-rs-sys/src/decompress.rs

```diff
@@ -34,11 +34,11 @@ macro_rules! decompress_stream {
                 ZSTD_getFrameContentSize($compressed.as_ptr() as *const c_void, $compressed.len());
 
             if size == ZSTD_CONTENTSIZE_UNKNOWN as _ {
-                panic!("ZSTD_CONTENTSIZE_UNKNOWN")
+                return Err("ZSTD_CONTENTSIZE_UNKNOWN");
             }
```

Comment:
what we can do instead here is provide a default size (e.g. 16kb) and have it decompress using that. Locally that looks like it works for me, and then returning a result is not needed.